Add de novo changelog (#185)
broadinstitute/gnomad-blog: content/changelog/2025-03-de-novo-variants-in-gnomad-v4-exomes.md

---
title: De novo variants in gnomAD v4 exomes
date: 2025-03-21
order: 1
---

> To expand the number of publicly available control *de novo* variants, we identified *de novo* variants (DNVs) in 1,517 releasable trios from gnomAD v4 exomes. We adapted Hail’s [hl.de_novo](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.de_novo) to align with [Kaitlin Samocha’s original de novo caller](https://github.com/ksamocha/de_novo_scripts/tree/master), incorporating PLs of the trios, a de novo prior probability, and a population frequency prior.

Reviewer: ```suggestion
To expand the number of publicly available control *de novo* variants, we have released 1,955 coding *de novo* variants (DNVs) called from 1,517 trios in the gnomAD v4.1 exomes.

<!-- end_excerpt -->
```
I'm not 100% sure we need the first part, but it's not a bad idea to give more context.

The next sentence is a bit in the weeds and should become the first sentence of the second paragraph with some rewording,
```
To generate these calls, we adapted Hail’s [hl.de_novo](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.de_novo) method and Kaitlin Samocha’s [de novo caller](https://github.com/ksamocha/de_novo_scripts/tree/master).
```
I don't think you need to mention
```
incorporating PLs of the trios, a de novo prior probability, and a population frequency prior.
```

> After applying a series of filters, we obtained a high-quality set of 1,955 coding DNVs. The observed de novo mutation rate per proband (~1.29 per exome) matches the expected rate.

Reviewer: can you briefly summarize the filters and define what high-quality is here? also, since this says "matches the expected rate", we should include a citation to note where that expected rate is from

Our team is currently working on implementing a more sophisticated algorithm developed by Mark Daly’s group. This approach will integrate a parent-specific de novo prior and account for false homozygous reference genotypes in the proband to further improve accuracy.

> For more details on our de novo detection methods, visit our [gnomad_qc GitHub repository](https://github.com/broadinstitute/gnomad_qc/blob/main/gnomad_qc/v4/create_release/create_de_novo_release.py).

Reviewer: ```suggestion
For more details on our de novo detection methods, visit the [gnomad_qc GitHub repository](https://github.com/broadinstitute/gnomad_qc/blob/main/gnomad_qc/v4/create_release/create_de_novo_release.py).
```
can you also link to the function in gnomad_methods?

> To download the DNV dataset, visit our [downloads page]().

Reviewer: can you reach out to the browser team to propose a name (e.g., something like https://gnomad.broadinstitute.org/data#v4-de-novo) and make sure it will be the correct URL here?

---
title: De novo variants from gnomAD v4.1 exomes
date: 2025-03-21
order: 1
---
To expand the number of publicly available control *de novo* variants, we have released 1,955 coding *de novo* variants (DNVs) called from 1,517 trios in the gnomAD v4.1 exomes.

> <!-- end_excerpt -->

Reviewer: ```suggestion
<!-- end_excerpt -->

```
I'm not 100% sure if you need the newline, but adding to be safe

> We generated these calls by adapting Hail’s [hl.de_novo](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.de_novo) method and Kaitlin Samocha’s [de novo caller](https://github.com/ksamocha/de_novo_scripts/tree/master). We filtered variants to include only variants that were outside low-confidence regions, did not have a `*` alt allele, passed variant QC, had coding consequences, and passed gnomAD v4.1 exomes allele frequency and callset allele count filters. We additionally filtered to keep only variants with high and medium confidence of being true *de novos*, which resulted in a high-quality set of 1,955 coding DNVs. The observed *de novo* mutation rate per proband (~1.29 per exome) aligns with expected rates. (Kaplanis & Samocha et al., Nature 2020).

Reviewer: one more nitpick -- can you add a hyperlink to the paper referenced here?
